template.yaml

```diff
@@ -981,4 +989,30 @@ Resources:
         - Key: Source
           Value: alphagov/di-auth-stub-relying-party
         - Key: Owner
-          Value: di-orchestration@digital.cabinet-office.gov.uk
\ No newline at end of file
+          Value: di-orchestration@digital.cabinet-office.gov.uk
+
+  #
+  # Secrets
+  #
+
+  IdentitySigningPublicKeySecret:
+    Type: AWS::SecretsManager::Secret
+    Properties:
+      Name: !Sub "/stubs/rp-${Environment}/IDENTITY_SIGNING_PUBLIC_KEY"
+      KmsKeyId: !GetAtt KmsKey.Arn
+
+  ECSSecretsManagerAccessManagedPolicy:
+    Type: AWS::IAM::ManagedPolicy
+    Properties:
+      PolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Effect: Allow
+            Action:
+              - secretsmanager:GetSecretValue
+            Resource: !Ref IdentitySigningPublicKeySecret
+          - Effect: Allow
+            Action: kms:Decrypt
+            Resource: !GetAtt KmsKey.Arn
+
+  # TODO ATO-159: Add client id and client private key once generated
```

Comment:
Should this TODO be committed?

Reply:
I've left it in so that I can go back to it as part of ATO-159 to make sure I add the correct secrets